test/integration/verify/controls/azurerm_virtual_machine_disk.rb

```diff
@@ -19,7 +19,7 @@
   end
 
   describe azurerm_virtual_machine_disk(resource_group: resource_group, name: unencrypted_disk_name) do
-    its('encryption_enabled') { should be false }
```

Comment:
perhaps the name field could be updated to: `name: attached_disk_name`

Reply:
This is be fixed now.